tests/fdb/test_fdb.py

```diff
@@ -95,14 +146,17 @@ def fdb_cleanup(ansible_adhoc, testbed):
     duthost = AnsibleHost(ansible_adhoc, testbed['dut'])
     try:
         duthost.command('sonic-clear fdb all')
+        time.sleep(5)
```

Comment:
Add a comment descirbing the issue you are W/A with sleep

Reply:
For ARP packets traffic is not verifying, so delays is redundant.

Reply:
Fixed.